hw/ip/test/src/tb_lib.hh

```diff
@@ -121,7 +121,12 @@ struct BootData {
     uint32_t core_count;
     uint32_t hartid_base;
     uint32_t tcdm_start;
-    uint32_t tcdm_end;
+    uint32_t tcdm_size;
+    uint32_t tcdm_offset;
+    uint32_t global_mem_start;
+    uint32_t global_mem_end;
+    uint32_t cluster_count;
+    uint32_t quadrant_count;
```

Comment:
I think just to be future proof we want to call this `s1_*` as in a full Manticore system there are more than S1 quadrants.
```suggestion
    uint32_t s1_quadrant_count;
```

Reply:
Makes sense